components/camel-jetty/src/test/java/org/apache/camel/component/jetty/JettyMuteExceptionTest.java

```diff
@@ -41,13 +41,29 @@ public void testMuteException() throws Exception {
         }
     }
 
+    @Test
+    public void testDefaultMuteException() throws Exception {
+        HttpGet get = new HttpGet("http://localhost:" + getPort() + "/fooDefault");
+        get.addHeader("Accept", "application/text");
+        try (CloseableHttpClient client = HttpClients.createDefault();
+             CloseableHttpResponse response = client.execute(get)) {
+
+            String responseString = EntityUtils.toString(response.getEntity(), "UTF-8");
+            assertEquals("", responseString);
```

Comment:
Maybe use `assertTrue(responseString.isEmpty())` ?

Reply:
Done, thanks for that.